primitives/sidechain-mc-hash/src/lib.rs

```diff
@@ -189,8 +189,10 @@ impl InherentDigest for McHashInherentDigest {
 	) -> Result<Self::Value, Box<dyn Error + Send + Sync>> {
 		for item in digest {
 			if let DigestItem::PreRuntime(id, data) = item {
+				let data =
+					data.clone().try_into().map_err(|_| "Invalid MC hash in digest: {data}")?;
```

Comment:
The error message then contains: `Invalid MC hash in digest: {data}`

Reply:
fixed